app/routes/endemics/which-species.js

```diff
@@ -17,17 +17,25 @@ const {
 const pageUrl = prefixUrl(endemicsWhichSpecies)
 const backLink = claimDashboard
 const errorMessage = { text: 'Select which species you are claiming for' }
-const view = `${endemicsWhichSpecies}`
 
 const getHandler = {
   method: 'GET',
   path: pageUrl,
   options: {
     handler: async (request, h) => {
-      const endemicsClaim = getEndemicsClaim(request)
-      return h.view(view, {
-        ...(endemicsClaim?.typeOfLivestock && {
-          previousAnswer: endemicsClaim.typeOfLivestock
+      // get type of livestock here, before we reset the session
+      const { organisation, typeOfLivestock } = getEndemicsClaim(request)
+
```

Comment:
Can you add the sbi to logging context here as we did before in the deleted handler?